src/less/modules/breadcrumbs.less

```diff
@@ -1,7 +1,9 @@
 .breadcrumb {
-  margin-left: 15px;
-  margin-top: -20px;
+  //TODO move breadcrumbs out into app-wide directive under navbar not within ui-view
```

Comment:
My knee-jerk reaction is to say that if we used ui-router for routing we'd be able to use something like https://github.com/ncuillery/angular-breadcrumb (or at least a similar approach).

we could move it out into the header service as it stands at the moment but I think it would be better to try and refactor the apps's routing in general and then work the breadcrumb fixes into that.